bHY6yh-d1e105/cod-pu6y7d_bHY6yh-d1e105.xml

```diff
@@ -162,11 +162,11 @@
             <lb ed="#V" n="66"/>ignitus.</quote> 
         </p>
         <p xml:id="bHY6yh-d1e105-d1e335"> ¶
-          Idem in <ref xml:id="bHY6yh-d1e105-Rd1e387">ecclesiastico. c. 8.</ref> <quote xml:id="bHY6yh-d1e105-Qd1e337" source="#http://scta.info/resource/qoh8_4" synch="2-6">
+          <ref Idem in xml:id="bHY6yh-d1e105-Rd1e387">ecclesiastico. c. 8.</ref> <quote xml:id="bHY6yh-d1e105-Qd1e337" source="#http://scta.info/resource/qoh8_4" synch="2-6">
```

Comment:
careful here. Your putting the text **inside** the element open tag instead of **between** the open and close tag. 
So instead it should look like the following.

```<ref xml:id="bHY6yh-d1e105-Rd1e387">Idem in ecclesiastico. c. 8.</ref>```